examples/hooks/components/layout/Tabs.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { cx } from '../../cx';
+
+import './Tabs.css';
+
+export type TabProps = {
+  title: string;
+};
+
+export const Tabs = ({ children }) => {
+  const [currentTab, setCurrentTab] = useState(0);
+
+  return (
+    <div className="ais-Tabs">
```

Comment:
The `<Tab>` component is not part of the InstantSearch.css spec so prefixing it with `ais` is confusing. The classname `.Tabs` should be fine. (Same for the inner components.)